chia/wallet/puzzles/tails.py

```diff
@@ -92,7 +92,9 @@ async def generate_issuance_bundle(cls, wallet, _: Dict, amount: uint64) -> Tupl
 
         inner_solution = wallet.standard_wallet.add_condition_to_solution(
             Program.to([51, 0, -113, tail, []]),
-            wallet.standard_wallet.make_solution(primaries=[Payment(cat_inner.get_tree_hash(), amount)]),
+            wallet.standard_wallet.make_solution(
+                primaries=[Payment(cat_inner.get_tree_hash(), amount, [cat_inner.get_tree_hash()])]
```

Comment:
`get_tree_hash()` isn't all that cheap. I think it would make sense to just call it once

Reply:
Its probably negligible here because it only happens when creating a CAT but i added 92d3dec25d4797b4eb133a3ed8f73beb5b3c47f5.